modules/ppcp-wc-gateway/services.php

```diff
@@ -815,7 +859,7 @@ static function ( $container ): AuthorizeOrderActionNotice {
 				),
 				'requirements' => array( 'messages' ),
 				'gateway'      => 'paypal',
-				'description'  => str_replace( '<a>', '<a href="https://www.paypal.com/us/business/buy-now-pay-later">', __( 'Customize the appearance of <a>Pay Later messages</a> on checkout to promote special financing offers, which help increase sales.', 'woocommerce-paypal-payments' ) ),
+				'description'  => str_replace( '<a>', '<a href="' . $messages_disclaimers->link_for_country() . '" target="_blank">', __( 'Displays Pay Later messaging for available offers. Restrictions apply. <a>Click here to learn more.</a>', 'woocommerce-paypal-payments' ) ),
```

Comment:
I think it shouldn't be like so. For the countries which shouldn't have restrictions, it displays the link with an empty 'href' attribute.

Reply:
@strangerkir Pay Later settings are only displayed on the countries that allows it: https://github.com/woocommerce/woocommerce-paypal-payments/blob/trunk/modules/ppcp-button/src/Helper/class-messagesapply.php#L23-L28

So for example if you set Store Address Country / State to "Spain" it does not show any Pay Later setting.

Reply:
Oh, you are right. 

But anyway, it gives me the <a> tag with the empty `href` attribute when the country is the UK.

Reply:
Good catch! :) I´ve just fixed now: https://github.com/woocommerce/woocommerce-paypal-payments/pull/142/commits/3d7fb7fc536871f70719f18502450093697dd41a?file-filters%5B%5D=.php&hide-deleted-files=true#diff-c20665203423aa892f69af009a83dc777d23631ec30ebb7776746ddb341a6f5fR28

when you have time please try it again and let me know, thanks.

Reply:
Thanks! Works well for me now.